src/main/java/kernbeisser/Windows/Supply/SupplySelector/SupplySelectorController.java

```diff
@@ -41,6 +48,35 @@ public void fillView(SupplySelectorView supplySelectorView) {
     loadDefaultDir();
   }
 
+  public void editArticle(LineContent lineContent) {
+    Supplier kkSupplier = Supplier.getKKSupplier();
+    switch (lineContent.getStatus()) {
+      case OK:
+      case ADDED:
+        try {
```

Comment:
Hier wird eine Maske aufgerufen mit dem Original Artikel? Um dann den Artikel zu bestätigen und dann die Daten wieder in den Line content zu übernehmen oder? Woher weiß der Nutzer das nur Veränderungen von den in refreshLineContent() genutzten Attributen übernommen werden?